apps/replicated-import/run.py

```diff
@@ -14,8 +14,9 @@
 def reset_schema(client: weaviate.Client, repl_factor: int):
     client.schema.delete_all()
     class_obj = {
-        "vectorizer": "none",
+        "vectorizer": "text2vec-bigram",
         "vectorIndexConfig": {
+            "sq": {"enabled": True, "trainingLimit": 10000, "rescoreLimit": 20},
```

Comment:
Hi @donomii, this should fail due to additional validation https://github.com/weaviate/weaviate/pull/5341. Both PQ and SQ require that either async indexing is enable or that they are enabled post loading some data. You can use [this import script](https://github.com/weaviate/weaviate-chaos-engineering/blob/73ecf9ac4a04fc6137b008b519a8d8cbf6bc5818/apps/ann-benchmarks/weaviate_import.py) on the other PR to show how importing is paused while PQ/SQ is enabled

Reply:
I added the ASYNC_INDEXING environment variable, since this seemed to make the most sense for the test.  So now the test confirms that a node can crash while objects are being queued for later indexing, and import still works.